Makefile

```diff
@@ -61,6 +62,22 @@ unlock-rpm:
 	sudo yum versionlock clear
 
 # Add other package types here.
+packages-deb:
```

Comment:
Please add a rule that generates mk_deb_fpmdir.stack_blackbox.txt so that changes to mk_rpm_fpmdir.stack_blackbox.txt are automatically copied to this package.

tools/mk_deb_fpmdir.stack_blackbox.txt: tools/mk_rpm_fpmdir.stack_blackbox.txt
  cp tools/mk_rpm_fpmdir.stack_blackbox.txt $@

See the rule for tools/mk_macports.vcs_blackbox.txt if you want to modify the file instead of copy it.